holisticai/privacy/attacks/__init__.py

```diff
@@ -0,0 +1 @@
+from ._BlackBoxAttack import BlackBoxAttack, append_if_not_empty
```

Comment:
Please add the docstring description in the init.py file similar to the below:

"""
The :mod:holisticai.privacy.metrics module includes classification, regression, multiclass, recommender and clustering privacy metrics
"""

Reply:
added.